packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -291,3 +304,21 @@ function dockerExec(args: string[], options?: SpawnSyncOptions) {
 
   return proc;
 }
+
+export function dockerCopyFromImage(image: string, imagePath: string, outputPath: string) {
```

Comment:
Why isn't this a method?

Reply:
I've now refactored this code into `BundlingDockerImage.copyOut`.